x-pack/plugins/enterprise_search/server/routes/app_search/curations.ts

```diff
@@ -67,6 +67,9 @@ export function registerCurationsRoutes({
     {
       path: '/api/app_search/engines/{engineName}/curations/{curationId}',
       validate: {
+        query: schema.object({
+          skip_record_analytics: schema.string(),
```

Comment:
not a blocker, just an aside: I wonder if we should formalize snake_casing vs camelCasing in API queries/payloads 🤔